ActiveLabel/ActiveLabel.swift

```diff
@@ -185,6 +185,10 @@ typealias ElementTuple = (range: NSRange, element: ActiveElement, type: ActiveTy
         didSet { textContainer.lineBreakMode = lineBreakMode }
     }
     
+    /// This is exposed as an internal var so that in UTs we can swap it for another implementation mock and stub it
+    var activeBuilder: ActiveBuilderInterface = ActiveBuilder(regexParser: RegexParser())
```

Comment:
why can't we inject it in a init method?

Reply:
for backwards compatibility - existing clients of this code are using built-in inits from UIKit such as `init(frame: CGRect)` and `init?(coder aDecoder: NSCoder)`, etc. We can't break their code by requiring it in init. Plus it doesn't seem like this is something that users of this library should be able to configure. That's why this property is not `open` or `public` but instead `internal`